src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.searchBarForm {
+  margin-top: 20px;
+  position: relative;
+
+}
+
+.searchBarBig {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
add font-family to html and use inherit for inputs

Reply:
tests don't work with html tag 
<img width="1057" alt="CleanShot 2022-08-16 at 15 09 14@2x" src="https://user-images.githubusercontent.com/103505670/184876211-91e0f548-52a5-46c8-a00d-252f1dc5d349.png">

I can use * { } and tests will be pass, but I think it's not good idea